setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(name='python-validity',
-      version='0.14',
+      version='0.15',
```

Comment:
pretty sure I don't need to change this
```suggestion
      version='0.14',
```